src/components/modalTabs/AposModalTabs.vue

```diff
@@ -3,52 +3,26 @@
     <ul class="apos-modal-tabs__tabs">
       <li class="apos-modal-tabs__tab" v-for="(group, i) in groups"
         :key="group.name">
-        <button :id="uid + i" class="apos-modal-tabs__btn" @click="selectTab"
-          :aria-selected="uid + i === current ? true : false"
+        <button :id="group.uid + i" class="apos-modal-tabs__btn" @click="selectTab"
+          :aria-selected="group.uid + i === current ? true : false"
         >{{ group.label }}</button>
       </li>
     </ul>
-    <form class="apos-modal-tabs__wrapper">
-      <fieldset v-for="(group, i) in groups" :key="group.name" :ref="uid + i"
-        :aria-labelledby="uid + i" class="apos-modal-tabs__pane"
-        :aria-hidden="uid + i === current ? false : true"
-      >
-        <!-- Temporary demo content: -->
-        <h2>Tab {{ i }}: {{ group.label }}</h2>
-        <div style="min-height: 70vh"></div>
-      </fieldset>
-    </form>
   </div>
 </template>
 
 <script>
 export default {
   name: 'AposModalTabs',
-  data () {
-    return {
-      uid: (Math.floor(Math.random() * Math.floor(10000))),
-      current: 0
-    }
-  },
   props: {
-    groups: Array
-  },
-  mounted: function () {
-    this.current = this.uid;
+    groups: Array,
```

Comment:
Yeah, this seems important to require, right?

Reply:
required